app/models/queue_item.rb

```diff
@@ -0,0 +1,22 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :video
+  belongs_to :user
+
+  def video_title
+    self.video.name
+  end
+
+  def rating
+    review = Review.where(user_id: user.id, video_id: video.id).first
```

Comment:
I know Kevin uses `.where(args).first` in solutions but this is Rails 3 thing. In Rails 4 - `.find_by` is preferred way to search for 1 record and `.where` for collection.